packages/devtools_app/lib/src/ui/vm_flag_widgets.dart

```diff
@@ -54,7 +55,8 @@ class ProfileGranularityDropdown extends StatelessWidget {
             Provider.of<BannerMessagesController>(context);
         if (safeValue == highProfilePeriod) {
           bannerMessageController.addMessage(
-            HighProfileGranularityMessage(screenId).build(context),
+            HighProfileGranularityMessage(screenId).build(context)
+                as BannerMessage,
```

Comment:
let's try to change the return type of HighProfileGranularityMessage to BannerMessage instead of Widget. Then we can get rid of this cast

Reply:
done